modules/win-autoscale/watchmaker-win-autoscale.template.cfn.yaml

```diff
@@ -548,6 +559,50 @@ Resources:
                       "log_stream_name": "default_logs_{instance_id}"
                     }
                   }
+            'c:\cfn\scripts\AmazonCloudWatchAgent\cloudwatch-applog-config.py':
+              content: !Sub 
+                - |-
+                  import json
+                  import os
+
+                  print('running cwl baseline modification script.')
+                  cloudwatch_baseline = 'c:/cfn/scripts/AmazonCloudWatchAgent/aws-cloudwatch-agent-config.json'
+                  log_group_name = "/aws/ec2/win/${AWS::StackName}"
+                  log_paths_input = (r'''${local_addCWLtarget}''').split(",")
```

Comment:
nit: ought to be able to use a raw `r` string in the lx templates also (minimize differences to simplify maintenance)

Reply:
updated